src/AppBundle/Twig/SourceCodeExtension.php

```diff
@@ -95,7 +95,7 @@ private function getTemplateCode()
 
     private function getTemplateRelativePath()
     {
-        return 'app/Resources/views/'.$this->template->getTemplateName();
+        return 'app'.DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'views'.DIRECTORY_SEPARATOR.$this->template->getTemplateName();
```

Comment:
Thanks for this information. I agree completely with you. However, we're lucky because this `show_source_code()` function is included only in the templates that we decide, so it doesn't have to work right for all the possible cases (at least for now).